frontend/src/global_styles/common/header/logo.sass

```diff
@@ -12,11 +12,15 @@
     background: no-repeat center
     background-size: contain
 
-  &-icon, &-icon_dark
-    margin-left: 0.5rem
-    height: 32px
-    background: no-repeat left center
-    background-size: contain
-
   @media screen and (max-width: 850px)
     display: none
+
+  &-icon, &-icon_white
```

Comment:
The [comment I made earlier](https://github.com/opf/openproject/pull/19711#discussion_r2239406922) is still true. These are the exact same syles that apply no matter whether you show the blue or the white logo. Imho, the reason you have to duplicate the class here, is that you don't apply the BEM prinicple correctly. The element would need the class `op-logo-icon` in any case and the white logo would get `op-logo-icon_white` additionally. Then the base class is responsible for the styling and the modifier takes care that the correct image is shown.
You'd need to change that in the `top_menu_helper.rb`